src/Button/Button.js

```diff
@@ -25,12 +27,19 @@ const Button = ({
         [`${styles.active} ${classes.active}`]: isActive,
         [styles.bordered]: border,
         [styles.fluid]: fluid,
-        [styles.disabled]: disabled
+        [styles.disabled]: disabled,
+        [styles.loading]: isLoading
       })}
       ref={forwardedRef}
       {...props}
     >
       {children}
```

Comment:
In the case of Loading, do we need to show children?

Reply:
yes, if I remove - width changed